andela/team-odd-bn-frontend
# #171177423  fixes a bug and will enable users to edit a one-way trip request

#### What does this PR do?
This PR fixes a bug and will enable users to edit a one-way trip request
#### Description of Task to be completed?
- In this PR, I changed the action file to ensure the return date field is removed from the data being sent to the backend.
#### How should this be manually tested?
- git clone `https://github.com/andela/team-odd-bn-frontend`
- git checkout `bg-fix-edit-onewaytrip-171177423`
- Install dependencies:  `npm i`
- Start the server: `npm run dev`
- Navigate to the `my Trips ` link on the sidebar.
- Choose a one-way trip  and click on view to see a single trip request details
-  Click edit and save the changes.

#### Any background context you want to provide?
The expectation is that the user is expected to edit a one-way trip request successfully. Previously, we were getting an error that the user has provided the wrong date. This happened because the return date was being sent to the back end for a one-way trip.
#### What are the relevant pivotal tracker stories?
[#171177423](https://www.pivotaltracker.com/n/projects/2407428/stories/171177423)
#### Screenshots (if appropriate)
<img width="1431" alt="Screen Shot 2020-02-10 at 17 42 53" src="https://user-images.githubusercontent.com/45129725/74164854-00454380-4c2d-11ea-83c1-cd7a97b24ad0.png">
<img width="1440" alt="Screen Shot 2020-02-10 at 17 43 13" src="https://user-images.githubusercontent.com/45129725/74164866-0509f780-4c2d-11ea-8803-925530a09544.png">

#### Questions:
